### ISSUE TITLE
No sound in iOS

### ISSUE THREAD

**Issue by rea-laura** [phetsims/friction#171]
<b>Test Device</b>

Tycho, Hopper

<b>Operating System</b>

iOS 9.3.5, iOS 12

<b>Browser</b>

Safari

<b>Problem Description</b>

For https://github.com/phetsims/QA/issues/276 sim has no sound. Confirmed in both iOS 9 and iOS 12. Sound works for other published sims, but not this one.

<b>Steps to Reproduce</b>

1) open sim
2) play with sim and note there is no sound.

**Comment by zepumph** [comment]
@jbphet over to you for this sound related issue. Let me know what I can do to help.

**Comment by jbphet** [comment]
This should be fixed and has been propagated to Friction's 1.5 release branch.  This problem was due to code that was recently added to defer the playback of sounds until the audio context changed to the "running" state.  There was platform-specific code for iOS that actually used the playing of a sound to cause the state change, but, since that playing is now deferred, it didn't work.  I've changed the iOS workaround code to explicitly call `audioContext.resume()`, which is actually simpler but I didn't know it would work for iOS.  It's nice that it does, but this should be thoroughly tested in the next RC.
